src/test/java/com/solvd/carina_testing/MobileTesting.java

```diff
@@ -0,0 +1,79 @@
+package com.solvd.carina_testing;
+
+import com.zebrunner.carina.core.AbstractTest;
+import com.zebrunner.carina.utils.mobile.IMobileUtils;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class MobileTesting extends AbstractTest implements IMobileUtils {
+
+    private static final String TITLE_TASK = "Go to the gym";
+    private static final  int INDEX_TASK = 2;
+
+    @Test
+    public void verifyCreatingTask(){
+        TodayPage todayPage =  new TodayPage(getDriver());
+        int numOfTasksBeforeAddNew = todayPage.getTaskCards().size();
+        todayPage.clickOnAddTaskButton();
+        Assert.assertTrue(todayPage.isAirplaneButtonPresent(), "Airplane button isn`t present!");
+        Assert.assertTrue(todayPage.isHideKeyboardPresent());
+        todayPage.createNewTask(TITLE_TASK);
+        int numOfTasksAfterAddNew = todayPage.getTaskCards().size();
+        Assert.assertTrue(numOfTasksAfterAddNew > numOfTasksBeforeAddNew, "New task was not added");
+    }
+
+    @Test
+    public void verifyDeleteTask(){
+        TodayPage todayPage = new TodayPage(getDriver());
+        Assert.assertTrue(todayPage.isTodayPageOpened(), "Today page is not opened!");
+        int numOfTasksBeforeDelete = todayPage.getTaskCards().size();
+        Assert.assertTrue(todayPage.getTaskCards().get(INDEX_TASK).isCheckBoxPresent(INDEX_TASK), "Check box is not present!");
+        todayPage.getTaskCards().get(INDEX_TASK).deleteTask(INDEX_TASK);
```

Comment:
when you fix those methods, you don't need indexes for isCheckBoxPresent and deleteTask